FinniversKit/Sources/Components/FrontPageSavedSearchesView/FrontPageSavedSearchesView.swift

```diff
@@ -0,0 +1,183 @@
+import UIKit
+
+public protocol FrontPageSavedSearchesViewDelegate: AnyObject {
+    func frontPageSavedSearchesView(_ view: FrontPageSavedSearchesView, didSelectSavedSearch savedSearch: FrontPageSavedSearchViewModel)
+    func frontPageSavedSearchesViewDidSelectActionButton(_ view: FrontPageSavedSearchesView)
+}
+
+public class FrontPageSavedSearchesView: UIView {
```

Comment:
FYI - This file isn't new. It's renamed from `FrontPageShelfView` - with modifications to remove favorites.